src/main/java/teammates/common/util/TimeHelper.java

```diff
@@ -217,6 +217,18 @@ public static String formatDate(Date date) {
         return sdf.format(date);
     }
 
+    /**
+     * Formats a date in the format D, dd MMM, yyyy. Example: Tue, 23 May, 2018
```

Comment:
Use the same date as below (5 May 2012) to better show the difference in display format.